tasks/ThrusterTask.cpp

```diff
@@ -100,11 +100,11 @@ void ThrusterTask::setGazeboModel( std::string const& pluginName, ModelPtr model
 {
     string worldName = GzGet((*(model->GetWorld())), Name, ());
 
-    string taskName = "gazebo::" + worldName + "::" + model->GetName() + "::" + pluginName;
+    string taskName = std::regex_replace(pluginName, std::regex("__"), "::");
```

Comment:
The name of the task is now part of the SDF definition. Use it directly.

To avoid changing the current interface, I would recommend adding a setGazeboPluginTaskName(string) to the plugin interface (ModelPluginTaskI).